.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Release with Maven
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "Default version to use when preparing a release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Default version to use for new local working copy."
+        required: true
+        default: "X.Y.Z-SNAPSHOT"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Configure Git User
+      run: |
+        git config user.email "actions@github.com"
+        git config user.name "GitHub Actions"
+    - name: Checkout code
+      uses: actions/checkout@v3.1.0
+    - name: Set up cache
+      uses: actions/cache@v3.0.11
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+    - name: Set up JDK 8
+      uses: actions/setup-java@v3
+      with:
+        java-version: 8
+        distribution: adopt
+        cache: maven
+        server-id: struts2-bootstrap.staging
```

Comment:
I think I used struts2-juery in the past also for the bootstrap plugin. Will it work with this?

Reply:
As far I know this should be in-sync with what you have in your [pom.xml](https://github.com/struts-community-plugins/struts2-bootstrap/blob/release/5.0.x/pom.xml#L27)